cmd/earthly/build_cmd.go

```diff
@@ -538,10 +540,29 @@ func (app *earthlyApp) actionBuildImp(cliCtx *cli.Context, flagArgs, nonFlagArgs
 	// Kick off logstream upload only when we've passed the necessary information to logbusSetup.
 	// This information is passed back right at the beginning of the build within earthfile2llb.
 	go func() {
+		beforeSelect := time.Now()
 		select {
 		case <-cliCtx.Context.Done():
+			if app.verbose {
+				now := time.Now()
+				app.console.Warnf(
```

Comment:
Should this be on for verbose output only?

Reply:
I think so

Reply:
Oh wait.. I now see the `app.verbose`. We have a `console.Verbosef` BTW, perhaps that's more appropriate. This works too though.

Reply:
using that now